WordPressKit/WordPressKit/ReaderTopicServiceError.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+
+public enum ReaderTopicServiceError: Error {
+    case invalidId
+    case topicNotfound(id: Int)
+    case remoteResponse(message: String?, url: String)
+    
+    public var description: String {
+        switch self {
+        case .invalidId:
+            return "Invalid id: an id must be valid or not nil"
```

Comment:
We still need to localize the string that's returned by passing it to `NSLocalizedString`.  There are lots of examples throughout the app to look at. :)   Be sure to include a helpful comment about how the string is used so our translators will understand its context. :)

One thing to be aware of is that any parameters included in the string will not have their final values when the string is translated.  In this case, I'd suggest maybe concatenating the parameter's value to the end of the concatenated string. 

No need to worry about the actual translation, we have a separate process that handles all string translations.